front/components/assistant_builder/types.ts

```diff
@@ -144,7 +146,8 @@ export function getDefaultAssistantState(): AssistantBuilderState {
       },
       temperature: 0.7,
     },
-  };
+    maxToolsUsePerRun: 3,
+  } satisfies AssistantBuilderState;
```

Comment:
Why did you need to change that?

Reply:
Because otherwise, the return value is just `AssistantBuilderState`, shadowing the fact that eg `maxToolsUsePerRun` is never null when coming out of this function

Reply:
So here we really want typescript to infer the proper return type of the function BUT we also want that type to satisfy `AssistantBuilderState`